packages/sitecore-jss-nextjs/package.json

```diff
@@ -66,7 +66,8 @@
     "typescript": "~4.9.4"
   },
   "peerDependencies": {
-    "@sitecore/engage": "^1.4.1",
+    "@sitecore-cloudsdk/events": "^0.1.1",
```

Comment:
Is `@sitecore-cloudsdk/events` a dependency for this package? I think just `@sitecore-cloudsdk/personalize`.

Reply:
@ambrauer  It was done in order to have /events and /personalized matched versions, otherwise there can be a mismatch

Reply:
Makes sense